Add platform button for time sync (#42873)
home-assistant/home-assistant.github.io: source/_integrations/bsblan.markdown

Reviewer: ```suggestion
  - Button 
  - Climate
```

The **BSB-Lan** {% term integration %} integrates [BSBLan](https://github.com/fredlcore/BSB-LAN) devices into Home Assistant.

BSBLan is a device that is made by `Frederik Holst` and with the help of many other contributors. The board v3 is designed for an Arduino Due with an Ethernet-Shield for web-based controlling of heating systems such as `Elco Thision`, `Brötje` and similar systems. Also available is an ESP32 version of the board.

It can interface with the heating system over Boiler-System-Bus, Local Process Bus, and <abbr title="Punkt-zu-Punkt Schnittstelle">PPS</abbr>. For more information on which systems it supports, take a look at their [documentation](https://docs.bsb-lan.de).

{% include integrations/config_flow.md %}

For authentication, HTTP authentication using a username and password or using a passkey is supported. Use either one.

## Supported functionality

Depending on your system, the following entities are available:

- Climate
- Water heater
- Button
- Diagnostics

> - Sensor

Reviewer: ```suggestion
- Button
- Climate
- Diagnostics
- Sensor
- Water heater
```

### Buttons

- Sync time

> Synchronizes the BSBLan device time with Home Assistant. Use this if your BSBLan device time drifts or does not match Home Assistant

Reviewer: ```suggestion
### Buttons

- **Sync time**: Synchronizes the BSB-Lan device time with the current Home Assistant time. Use it when your device's time drifts or doesn't match Home Assistant's time.

The **Sync time** button appears under the **Configuration** section of the device page, not on your dashboards by default. You can also trigger the same synchronization programmatically using the `bsblan.sync_time` action, for example, in a daily automation.
```
not 100% sure about this.

The **BSB-Lan** {% term integration %} integrates [BSBLan](https://github.com/fredlcore/BSB-LAN) devices into Home Assistant.

BSBLan is a device that is made by `Frederik Holst` and with the help of many other contributors. The board v3 is designed for an Arduino Due with an Ethernet-Shield for web-based controlling of heating systems such as `Elco Thision`, `Brötje` and similar systems. Also available is an ESP32 version of the board.

It can interface with the heating system over Boiler-System-Bus, Local Process Bus, and <abbr title="Punkt-zu-Punkt Schnittstelle">PPS</abbr>. For more information on which systems it supports, take a look at their [documentation](https://docs.bsb-lan.de).

{% include integrations/config_flow.md %}

For authentication, HTTP authentication using a username and password or using a passkey is supported. Use either one.

## Supported functionality

> 

Reviewer: ```suggestion

Depending on your system, the following entities are available:

- Climate
- Water heater
- Button
- Diagnostics
- Sensor
```
I would move this generic information into the introduction of the section

### Buttons

- Sync time

Synchronizes the BSBLan device time with Home Assistant. Use this if your BSBLan device time drifts or does not match Home Assistant

### Sensors

The following sensors are available, depending on your heating system:

- Inside temperature
- Outside temperature

### Platforms

The following platforms are available, depending on your system:

- Climate
- Water heater
- Button
- Diagnostics

> - Sensor

Reviewer: ```suggestion
```

The **BSB-Lan** {% term integration %} integrates [BSBLan](https://github.com/fredlcore/BSB-LAN) devices into Home Assistant.

BSBLan is a device that is made by `Frederik Holst` and with the help of many other contributors. The board v3 is designed for an Arduino Due with an Ethernet-Shield for web-based controlling of heating systems such as `Elco Thision`, `Brötje` and similar systems. Also available is an ESP32 version of the board.

It can interface with the heating system over Boiler-System-Bus, Local Process Bus, and <abbr title="Punkt-zu-Punkt Schnittstelle">PPS</abbr>. For more information on which systems it supports, take a look at their [documentation](https://docs.bsb-lan.de).

{% include integrations/config_flow.md %}

For authentication, HTTP authentication using a username and password or using a passkey is supported. Use either one.

## Supported functionality

Depending on your system, the following entities are available:

- Climate
- Water heater
- Button
- Diagnostics

> - Sensor

Reviewer: ```suggestion
- Sensor

```

The **BSB-Lan** {% term integration %} integrates [BSBLan](https://github.com/fredlcore/BSB-LAN) devices into Home Assistant.

BSBLan is a device that is made by `Frederik Holst` and with the help of many other contributors. The board v3 is designed for an Arduino Due with an Ethernet-Shield for web-based controlling of heating systems such as `Elco Thision`, `Brötje` and similar systems. Also available is an ESP32 version of the board.

It can interface with the heating system over Boiler-System-Bus, Local Process Bus, and <abbr title="Punkt-zu-Punkt Schnittstelle">PPS</abbr>. For more information on which systems it supports, take a look at their [documentation](https://docs.bsb-lan.de).

{% include integrations/config_flow.md %}

For authentication, HTTP authentication using a username and password or using a passkey is supported. Use either one.

## Supported functionality

Depending on your system, the following entities are available:

- Button
- Climate
- Diagnostics
- Sensor
- Water heater

> ### Buttons
> 

Reviewer: ```suggestion
```
